internal/sessionmanager/sessionmanager.go

```diff
@@ -143,47 +131,3 @@ func (sm *SessionManager) GetNextSessionID() uint64 {
 	sm.sessID++
 	return sm.sessID
 }
-
-// ReceiveFrom is called when a new message is received
-func (sm *SessionManager) ReceiveFrom(ctx context.Context, p peer.ID, blks []cid.Cid, haves []cid.Cid, dontHaves []cid.Cid) {
```

Comment:
The WantRequestManager now takes care of receiving incoming messages and distributing them to sessions